workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/service/JiraService.java

```diff
@@ -424,6 +589,58 @@ private JSONObject _search(
 		return null;
 	}
 
+	private JSONObject _search(
+			String jql, int maxResults, String[] returnFields, int startAt)
+		throws Exception {
+
+		return _search(jql, maxResults, null, returnFields, startAt);
+	}
+
+	private JSONArray _transformADFTextArea(String text) {
+		JSONObject textJSONObject = new JSONObject(
```

Comment:
can you complress and inline the jsonObjects like here?
https://github.com/liferay/liferay-portal/blob/dab63911b779b5e949b3037e0036edd4940f290a/workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/TicketAttachmentsCompleteUploadRestController.java#L144

Reply:
Also inline other places too.